src/api/app/components/workflow_run_filter_input_component.html.haml

```diff
@@ -0,0 +1,7 @@
+= form_tag(token_workflow_runs_path(token_id), method: :get, class: 'd-flex px-3 py-2') do
```

Comment:
I'd rather stick to the same appearance.
Maybe you can still give the `.list-group-item` CSS class to this element to be consistent with the other filters, and give the `.active` CSS class in case it is selected. It will behave the same way of the others by having a green background when in use.
For the button instead keeping the white background should be enough in both active/non active cases.